changelogs/fragments/1880-fix_cobbler_system_ssl.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - fix cobbler_system ssl cert checkwhen validate_certs set to no
```

Comment:
```suggestion
  - cobbler_sync, cobbler_system - fix SSL/TLS certificate check when ``validate_certs`` set to ``false`` (https://github.com/ansible-collections/community.general/pull/1880).
```

Reply:
Renamed and squashed into a single commit.